### PR TITLE
Integration tests fixing alt home

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
Fixing the problems found in https://github.com/owncloud/core/issues/26918

Running tests locally, will add output after they finish.

### CODE REVIEW

File: build/integration/features/bootstrap/WebDav.php [hunk 1]
```diff
@@ -456,6 +456,17 @@ public function userUploadsAFileTo($user, $source, $destination)
 	}
 
 	/**
+	 * @When User :user uploads file from his home :source to :destination
+	 * @param string $user
+	 * @param string $source
+	 * @param string $destination
+	 */
+	public function userUploadsAFileFromHisHomeTo($user, $source, $destination)
```
Comment (PVince81) [inline build/integration/features/bootstrap/WebDav.php]: with "home" do you mean local folder on the local machine, not ownCloud home ?
This wording is confusing.

**Review by PVince81 — COMMENTED** [review]
Looks good except for the wording issue

File: build/integration/features/bootstrap/WebDav.php (hunk 1 above)
Comment (SergioBertolinSG) [inline build/integration/features/bootstrap/WebDav.php]: No, I mean owncloud home. User is also an owncloud user.

Comment (PVince81) [inline build/integration/features/bootstrap/WebDav.php]: This scenario as worded is not possible in a regular ownCloud. You can't upload a file from your ownCloud home. You can upload **to** your ownCloud home.

I think the reason for this is that you're reading the file directly from inside the data folder to test upload ? You should rather read the file from an external folder containing sample files.

Comment (PVince81) [inline build/integration/features/bootstrap/WebDav.php]: Also one problem with reading a file directly from the data folder is that in some cases that file could be encrypted, that could break tests.

**Comment by SergioBertolinSG** [comment]
Tests using alt home variable have passed.

File: build/integration/features/bootstrap/WebDav.php (hunk 1 above)
Comment (SergioBertolinSG) [inline build/integration/features/bootstrap/WebDav.php]: OK, then I'll have to change those scenarios.

Comment (SergioBertolinSG) [inline build/integration/features/bootstrap/WebDav.php]: Moving the file seems more appropiate.

Comment (SergioBertolinSG) [inline build/integration/features/bootstrap/WebDav.php]: I have changed the federation tests, now that part is not needed.

**Review by PVince81 — APPROVED** [review]

**Comment by SergioBertolinSG** [comment]
Tests have passed locally after last changes.

**Comment by PVince81** [comment]
Thanks, merging then 👍 

I'll then rebase the other PR after that.

**Pull request merged**